nix/tests/postgresql.conf.in

```diff
@@ -718,7 +718,7 @@ default_text_search_config = 'pg_catalog.english'
 
```

Comment:
we actually no longer use this file in either `checks` or `start-server` so if you have a change you want to use or test like this you should now just make the change in `/ansible/files/postgresql_config/postgresql.conf.j2` and that is literally now the file that is loaded in `nix build .#checks ...` and `nix run .#start-server ...` and `nix run .#dbmate-tool ...`

Reply:
@soedirgo sorry I may have been wrong about this, because I think in pgtap we want a different more "bare" pgconf

```
initdb --locale=C --username=supabase_admin -D "$PGTAP_CLUSTER"
substitute ${./nix/tests/postgresql.conf.in} "$PGTAP_CLUSTER"/postgresql.conf
```

where we just unit test the sql as it is written in those tests. If you agree, we can keep this, or we can refactor it to use the config you think we should use specifically for pgtap tests.

Reply:
Would prefer to deduplicate, but if it's too much work let's just take the easy way out 👍